### PR TITLE
Add sparda parser

### PR DESCRIPTION
**Pull request by an-dr-eas-k**
this pull request adds the single feature to parse sparda csv files.

### CODE REVIEW

File: bankcsvtoqif/banks/sparda.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        val = self.get_amount(line[3])
+        return val if val >= 0 else 0
+
+    def get_target_account(self, line):
+        if len(line) == 6:
+           s = line[5]
+        else:
+           s = self.default_target_account;
+        return s
+
```
Comment (niknow) [inline bankcsvtoqif/banks/sparda.py]: Does that last function not belong to the feature pull request?

File: bankcsvtoqif/tests/banks/test_db_giro.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ class TestDBGiro(unittest.TestCase):
     def setUp(self):
         self.csv = """22.04.2015;22.04.2015;"SEPA-Überweisung an";\
         Smith, John;Rent;DE12345678909876543212;\
-        BYLADEM1GLA;;;;;;;-10,00;;EUR"""
+        BYLADEM1GLA;;;;;;;;-10,00;;EUR"""
```
Comment (niknow) [inline bankcsvtoqif/tests/banks/test_db_giro.py]: Why does the addition of sparda require a change in the unit tests of db_giro?

File: bankcsvtoqif/tests/test_transaction.py [hunk 3]
```diff
@@ -39,15 +39,15 @@
...
+    ['25.07.2015', '26.07.2015', '', '', 'description1', '', '', '', '', '', '', '', '', '', '-1,23', '', 'EUR'],
+    ['27.07.2015', '28.07.2015', '', '', 'description2', '', '', '', '', '', '', '', '', '', '-2,34', '', 'EUR'],
+    ['28.07.2015', '29.07.2015', '', '', 'description3', '', '', '', '', '', '', '', '', '', '', '4,56', 'EUR'],
     ['balance', '28.07.2015', '', '', '2.345,67', 'EUR', ],
 ]
 
 csv_line = """22.04.2015;22.04.2015;"SEPA-Überweisung an";\
         Smith, John;Rent;DE12345678909876543212;\
-        BYLADEM1GLA;;;;;;;-10,00;;EUR"""
+        BYLADEM1GLA;;;;;;;;-10,00;;EUR"""
```
Comment (niknow) [inline bankcsvtoqif/tests/test_transaction.py]: As above, why are changes required in this file?

**Comment by niknow** [comment]
Very good, thanks! I just have some minor remarks (see notes at the source code).

File: bankcsvtoqif/banks/sparda.py (hunk 1 above)
Comment (an-dr-eas-k) [inline bankcsvtoqif/banks/sparda.py]: yes, your right. I was lazy when sorting out the changes.

File: bankcsvtoqif/tests/banks/test_db_giro.py (hunk 2 above)
Comment (an-dr-eas-k) [inline bankcsvtoqif/tests/banks/test_db_giro.py]: It is not required, but fixes the current build errors on  travis

File: bankcsvtoqif/tests/test_transaction.py (hunk 3 above)
Comment (an-dr-eas-k) [inline bankcsvtoqif/tests/test_transaction.py]: It is not required, but fixes the current build errors on  travis

**Comment by niknow** [comment]
Nicely done, thanks!

**Pull request merged**